src/components/PathwaysList/PathwaysList.tsx

```diff
@@ -19,6 +19,7 @@ import {
 
 interface PathwaysListElementProps {
   evaluatedPathway: EvaluatedPathway;
+  criteria: CriteriaResult | undefined;
```

Comment:
Not a big deal but this can be changed to 
```suggestion
  criteria?: CriteriaResult;
```